src/CloudVariable/Encoder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\CloudVariable;
+
+/**
+ * Encodes data into Cloud format.
+ */
+class Encoder implements EncoderInterface
```

Comment:
You don't use this class. Add usages or remove it

Reply:
Ok, will remove it